### DISCUSSION TITLE
Implementation of Profiles

### DISCUSSION THREAD

**Discussion by floydtree** [ocsf/ocsf-schema#278] [Polls]
See the discussion - https://github.com/ocsf/ocsf-schema/discussions/277

**Comment by rroupski** [comment]
There are 3 cases:

1.  No profiles -- profiles=[]
2. A list of profiles  profiles=["host", "user"]
3. All profiles?

This vote covers only the first 2 cases.

**Reply by floydtree** [reply]
The aim of this poll is to decide if `metadata.profiles` should be an optional field. Let's discuss your point here https://github.com/ocsf/ocsf-schema/discussions/277
